unit10/unit10.js

```diff
@@ -0,0 +1,145 @@
+//Bai : 1  in một bảng của một số bất kì với các số từ 1 đến 10 và hiển thị ra kết quả 
+const tableMultiplication = (number) => {
```

Comment:
Khi dùng arrow function, nếu chỉ có 1 param duy nhất thì có thể bỏ dấu `()` đi
```suggestion
const tableMultiplication = number => {
```